sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightService.cs

```diff
@@ -145,6 +145,7 @@ public async Task InitializeAsync(CancellationToken cancellationToken = default)
             // Since playwright-dotnet checks PLAYWRIGHT_SERVICE_ACCESS_TOKEN and PLAYWRIGHT_SERVICE_URL to be set, remove PLAYWRIGHT_SERVICE_URL so that tests are run locally.
             // If customers use GetConnectOptionsAsync, after setting disableScalableExecution, an error will be thrown.
             Environment.SetEnvironmentVariable(ServiceEnvironmentVariable.PlaywrightServiceUri, null);
+            return;
```

Comment:
should we add explicit test for this scenario to test in unit test ?

Reply:
already added, there was an existing test to check rotation timer was setup in case scalable is disabled. That test has been modified to check this will return and rotation timer is null.